### ISSUE TITLE
Mint shouldn't require the parameter command quoted

### ISSUE THREAD

**Issue by toshi0383** [yonaskolb/Mint#39]
# Current behavior
```
Mint $ ./.build/x86_64-apple-macosx10.10/debug/mint run toshi0383/cmdshelf cmdshelf run hello.sh
🌱  The operation couldn’t be completed. (Mint.CommandError error 2.)
Mint $ ./.build/x86_64-apple-macosx10.10/debug/mint run toshi0383/cmdshelf "cmdshelf run hello.sh"
🌱  Finding latest version of cmdshelf
🌱  Using cmdshelf "0.8.0"
🌱  cmdshelf "0.8.0" already installed
🌱  Running cmdshelf "0.8.0"...
hello
```

# Expected Behavior
```
Mint $ ./.build/x86_64-apple-macosx10.10/debug/mint run toshi0383/cmdshelf cmdshelf run hello.sh
🌱  Finding latest version of cmdshelf
🌱  Using cmdshelf "0.8.0"
🌱  cmdshelf "0.8.0" already installed
🌱  Running cmdshelf "0.8.0"...
hello
Mint $ ./.build/x86_64-apple-macosx10.10/debug/mint run toshi0383/cmdshelf "cmdshelf run hello.sh"
🌱  Finding latest version of cmdshelf
🌱  Using cmdshelf "0.8.0"
🌱  cmdshelf "0.8.0" already installed
🌱  Running cmdshelf "0.8.0"...
hello
```

# Why
- Because it's shorter.
- So you can create an alias on that command like below, and **pass arguments to it**.

```
$ alias sourcery-0.10.0='mint run krzysztofzablocki/Sourcery@0.10.0 sourcery' # not actually works I guess, but just for example
$ sourcery-0.10.0 --help
```

**Comment by yonaskolb** [comment]
That would be nice, but if there are flags there's no way to know if there are from mint or from the command that is being run

**Comment by yonaskolb** [comment]
I guess if there are no flags it would be possible though

**Comment by toshi0383** [comment]
What do you use for command-line parser?
I think it's possible if you do manual arguments parsing, which I'm doing in my fork of Commander linked from `cmdshelf`👌.

**Comment by toshi0383** [comment]
Actually I'm doing manual parsing inside `cmdshelf`. I just changed Commander's `func shift()` to treat every arguments as plain Strings, and supressed `unknown argument` error on uncompleted parsing state.

**Comment by yonaskolb** [comment]
I was originally using Commander (which would actually throw an error if you passed an unexpected flags), but have been using Guaka for a while.

2 notes:
- I fixed the error above in master. It was a regression above. It now again prints "Too many arguments. Make sure command is surrounded in quotes"
- When using swift PM you can use `swift run` instead of looking in the build folders: `swift run mint run toshi0383/cmdshelf`

**Comment by toshi0383** [comment]
> swift run mint run toshi0383/cmdshelf

Nice, so it would be like `swift run mint run toshi0383/cmdshelf 'cmdshelf run run.sh'`.
That's a lot of runs!🤣

**Comment by yonaskolb** [comment]
The fact that Swift PM supports this non quoted behaviour as well, makes me want this more 👍

**Comment by yonaskolb** [comment]
This is solved in #43 

This now works when running from the Mint repo:
```sh
swift run mint run toshi0383/cmdshelf cmdshelf run hello.sh
```
